modules/nupac.nu

```diff
@@ -302,9 +287,17 @@ export def "nupac search" [
     # Search for package named example
     #> nupac search example
 ] {
-    get-repo-contents
-    |where name =~ $query or short-desc =~ $query or long-desc =~ $query or keywords =~ $query
-    |reject script-url script-raw-url
+    let contents = (
```

Comment:
unnecessary state

Reply:
Then how should I display list contents in such case?
![image](https://user-images.githubusercontent.com/76530971/176632757-55f6a8a5-49be-4a0d-a53e-cc2192d3ba66.png)

Reply:
`str collect ", "` doesn't work 
![image](https://user-images.githubusercontent.com/76530971/176633173-59ab6856-a5bc-4d8f-91d8-d14546834e50.png)